draft-sharabayko-mops-srt.md

```diff
@@ -1178,6 +1217,97 @@ Destination Socket ID: 32 bits.
 
 ACKACK control packets do not contain Control Information Field (CIF).
 
+### Message Drop Request {#ctrl-pkt-dropreq}
+
+The Message Drop Request is sent by the sender to the receiver
+in case of some internal error, when an unacknowledged packet is not present
+in sender's buffer. Thus the sender notifies the receiver that it must not
+wait for retransmission of this message.
+Note the Message Drop Request is not sent if sender decides to drop a message
+according to the TL Packet Drop mechanism, as in this case the receiver
```

Comment:
"TL Packet Drop" first appeared here. For the term consistency, it would be better to be "Too-Late Packet Drop" or "TLPKTDROP".